setup.py

```diff
@@ -25,7 +25,6 @@
       classifiers=[
           'Framework :: Plone',
           'Framework :: Plone :: 4.0',
```

Comment:
Could you please also remove the Plone 4.0 classifier too? We slightly hijack this pull request for this, so we should do it an separate commit 😉